frameworks/thorntail/src/main/java/io/dekorate/thorntail/configurator/ThorntailPrometheusAgentConfigurator.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2019 The original authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.dekorate.thorntail.configurator;
+
+import io.dekorate.kubernetes.config.Configurator;
+import io.dekorate.openshift.config.OpenshiftConfigFluent;
+
+import java.util.Arrays;
+
+/**
+ * For OpenShift deployments, we typically use a base image that is similar to Fabric8 image.
+ * It contains a Prometheus agent which messes up the java.util.logging configuration, so we switch it off here.
+ * See <a href="http://issues.jboss.org/browse/THORN-1859">THORN-1859</a> for more details.
+ */
+public class ThorntailPrometheusAgentConfigurator extends Configurator<OpenshiftConfigFluent<?>> {
```

Comment:
Oh and one more question. I made this `Configurator<OpenshiftConfigFluent<?>>`, so it only applies to OpenShift resources and doesn't apply to Kubernetes resources. Is that a good idea? Or should that be `Configurator<BaseConfigFluent<?>>` so that it applies to Kubernetes resources too?

Reply:
A question for the master @iocanel ^

Reply:
@Ladicek since this configurator is adding an environmnet variable, which is soemthing applicable to both kubernetes and openshift I would go with `BaseConfigFluent`.

Reply:
OK, I'll change that.

Reply:
Fixed.